app/views/reading_collections/_new.html.erb

```diff
@@ -0,0 +1,14 @@
+<% title "New Reading Collection" %>
+
+<h3>New Reading Collection</h3>
+<%= form_tag "/reading_collections", method: :post do %>
+  <%= label_tag :name %>
+  <%= text_field_tag :name %>
+    <br>
+  <% @tags.each do |tag| %>
+    <%= check_box_tag "tags[]", tag.name %>
+    <%= h tag.name %>
+      <br>
+  <% end %>
+  <%= submit_tag "Create Collection" %>
```

Comment:
If we're building off what we discussed yesterday, I think we'll need to change this to just have the div so I can add the CollectionForm component via pack here. I can modify that, just want to check that that's cool.

Reply:
Ah, Phil can chime in here but we actually won't need to use this at all.

Reply:
Yea, let's still make the changes we talked about. I left those just so we could test, but let's pair and we can knock this out tomorrow.

Reply:
This partial will be removed on Joel's current brach that will be PRd next.